azure-pipelines.yml

```diff
@@ -16,10 +16,19 @@ variables:
     value: true
   - name: _BuildConfig
     value: 'Debug'
-  - name: Codeql.Enabled
-    value: true
-  # used for post-build phases, internal builds only
   - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+    - name: Codeql.Enabled
```

Comment:
Not sure how big a deal it is but will get `CodeQL3000` tasks auto-injected into all build jobs. That's at best redundant.